src/services/utxoSumForAddress.ts

```diff
@@ -6,7 +6,9 @@ export const askUtxoSumForAddresses = async (pool: Pool, addresses: string[]): P
     const sqlQuery = `
     SELECT "public"."utxo_view"."value" AS "value"
     FROM "public"."utxo_view"
+    LEFT JOIN "tx" ON "utxo_view"."tx_id" = "tx"."id"
     WHERE "public"."utxo_view"."address" = any(($1)::varchar array)
+    AND ("tx"."valid_contract" OR "tx"."id" IS NULL)
```

Comment:
@pedromtcosta, you have to use same utxo filtering logic here as in utxo-for-address selection, because the sum should be calculated from the same exact set of utxos that is being returned from the other endpoint

Reply:
@vsubhuman I changed this query to be the same as the one we have on the `/txs/utxoForAddresses` endpoint, with the only difference that the summing query ofc runs a `SUM` on value. We might want to create a view for this query as well, or change the `utxo_view` to use this query. What do you think?

Reply:
@vsubhuman see [this commit](https://github.com/Emurgo/yoroi-graphql-migration-backend/pull/261/commits/59e98e2ea6dde3ba0257d64f64ee982cf5ade682) - it adds the view with the filtering logic from the query above. I tested with some different addresses and I got the expected results in both endpoints. Summing the amounts from `/txs/utxoForAddresses` manually also yields the same result as we get from `/txs/utxoSumForAddresses`